lfs/config.go

```diff
@@ -134,11 +141,10 @@ func (c *Configuration) BatchTransfer() bool {
 		}
 
 		// Any numeric value except 0 is considered true
-		if n, err := strconv.Atoi(v); err == nil && n != 0 {
-			return true
-		}
+		n, _ := strconv.Atoi(v)
+		return n != 0
```

Comment:
You need to handle the "false" case here (to replace the "true" case above), only lfs.batch=0 will work to disable otherwise.